<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Word_Processingclass</class>
 <widget class="QWidget" name="Word_Processingclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word_Processingclass</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1173</width>
        <height>808</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="8" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_3"/>
       </item>
       <item row="9" column="3">
        <widget class="QPushButton" name="change_pushButton">
         <property name="text">
          <string>修改以显示效果</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>输入文字：</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="10" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="3">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QComboBox" name="thickness_comboBox">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="fontFace_comboBox_2">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>simplex</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>plain</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>duplex</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>complex</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>triplex</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>complex_small</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>script_simplex</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>script_complex</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="fontScale_comboBox_3">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>笔画粗细：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_9">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>字体：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>字号：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="7" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="frame_checkBox">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>是否添加方框</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="underline_checkBox">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>是否添加下划线</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>预览效果</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QTextEdit" name="textEdit">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="title">
          <string>字体颜色</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>161</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QSlider" name="red_horizontalSlider">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="green_horizontalSlider">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="blue_horizontalSlider">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>21</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>红</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>绿</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>蓝</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QTextEdit" name="textEdit_2">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>20</y>
            <width>61</width>
            <height>91</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>图片位置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="x_img_location">
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="y_img_location">
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>图片大小：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>长</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_img_longth">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>1000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>宽</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_img_width">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>1000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1" rowspan="10">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>img</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>red_horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Word_Processingclass</receiver>
   <slot>on_Slider_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>874</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>1072</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>green_horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Word_Processingclass</receiver>
   <slot>on_Slider_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>947</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>1069</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blue_horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Word_Processingclass</receiver>
   <slot>on_Slider_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>912</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>1072</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>change_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Word_Processingclass</receiver>
   <slot>on_Btnchange_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>968</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>1123</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frame_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Word_Processingclass</receiver>
   <slot>on_chkBoxframe_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>849</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>1194</x>
     <y>624</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>underline_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>Word_Processingclass</receiver>
   <slot>on_chkBoxunderline_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1083</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>1191</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_Slider_valueChanged(int)</slot>
  <slot>on_Btnchange_clicked()</slot>
  <slot>on_chkBoxframe_clicked(bool)</slot>
  <slot>on_chkBoxunderline_clicked(bool)</slot>
 </slots>
</ui>
